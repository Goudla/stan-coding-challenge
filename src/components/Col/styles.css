.col {
  width: 13%;
  margin-bottom: 30px;
}

@media (max-width: 980px) {
  .col {
    width: 19%;
  }
}

@media (max-width: 736px) {
  .col {
    width: 32%;
  }
}

@media (max-width: 480px) {
  .col {
    width: 49%;
  }
}
