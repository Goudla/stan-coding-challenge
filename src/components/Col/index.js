import * as React from 'react';

// Styles
import './styles.css';

type Props = {
  children: React.Node
};

class Col extends React.PureComponent<Props> {
  render() {
    const { children } = this.props;

    return (
      <div className="col">
        {children}
      </div>
    );
  }
}

export default Col;
