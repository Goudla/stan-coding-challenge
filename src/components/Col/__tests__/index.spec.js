import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import Col from '../';

describe('<Col />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Col>Test</Col>);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
