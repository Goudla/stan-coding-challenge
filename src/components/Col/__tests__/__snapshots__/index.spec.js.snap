// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Col /> Should render 1`] = `
<div
  className="col"
>
  Test
</div>
`;
